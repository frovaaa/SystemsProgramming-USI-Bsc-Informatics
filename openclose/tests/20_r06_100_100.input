bpqdpqbdb4pbdbdppbdqpqqbdq5ppqqbbdd4pqbpb6bpq7bdbbpqbddppbppqbbdbdpqpqdpqpqpqqbpqdbbdddqqdddqdd
bdpqp4bpbpqpbb1d5dpbppp9pqpqpbbbpqpbbdpppqqpqpqppqbbpqpb9dqbbdbpqpppqpqqqddddqqdqdddqqqqdqqdqdq
pbdqpppb7pbpbbddpbpppqqppqqqpppbbdbdbppbpbdbdqdbdqb9dbpbpbbbdpbbpqpqddppqpqqqddqdqdqddqqqdqqdqdqqq
bdpqpppb9pbb3dppqpqbpbppbbddpqbd0pbppqbpqppqpqpbpqb1pqb7pbdqppbdpqbpqdqq1dddqqdqdqqqdqdqdqdqqq
b5bbpbbbdpqpbbpbbbdppbbpbbbbdppqqddppp0pqqpqbbddppqqqbbbpbbdpbbb8pq7dddqdqdddqdqddqqddqddqddqddd
p8qpqbbppbbbbppppqpqpppbdppqbpqbp1bpqpqbbbpbpqppbdbbpbbpbbdbpqpqddqddqddqqdqddddqddqqqqqqqddddqqdd
bpqbpbdqpbbdbddpqbdqdpbdbdbdbpqb9bdddbdqpbbbd8dpbbppbpbdqpbbdbbbpbbbbbbbbddpqddddddqddddqdqqddqdqd
ppqbpbbbbpqbdddbpbbbpbppqpqqdqpqddbdpbdqbbbppbp2b2pqdpb2bddpbbddqbdpbbddqbdbdpqqbdqdqqddddqdddqdq
bbbddpbbbbppbbbdbpbppqqdqd6ppqbpqpppbdpqqqpbd69bpbppqbdbp32pbpbb3bbddddqdqqdqdqdqqdqddqqddddqd
pbdppppqbppbbdbppqqbppbpqbpqbpqpqbb8ppq7qpqbddbdpppqbbbpqddppqpqqdbbp5pqppqqqddqqddddqqdddqqdqqqq
ppqpbdpbpqdbdpbpqdqqqpppqpbdbb4ppqbpbdqb4pqbpbbpbbddb0bpqdbpqdppqppqqbdppbdpbdqqqqdqddqdddqddqqqq
bpp0bppbpqpqppqpb2b9pbdpbdqb2pbpqdqdpqqbdpqbpb9pqdbp0qbddqpbbbpbpbdqdbpbpqdqdqdddqdddqqdqqdqqd
bpqdpbdpbbbpp8bpppqppbdp2bdp3qqqppq3pbpbdqdppbdppp2pq9pbdbd9bppqp3bbddqqdqqqqqqqqqqqdqqdddqq
pqbpb4pqppqbd3bd2bppbpqpp2bpqbbbpbpbpqdppp2bdpqbbpqbdbpbdqpqpbdqbbd9pqddddqqqqdqddddqqdqqdqdqd
ppqqbdbpbpqb0dpbdbbbpbpqpqbddbpbbbb8bpqpqd4pqdpqp3pbbdb8bbpbbpqdd8bpqdpqpqqddddqqdddqdqdddqdqd
pbbddpbbbpbp0bdqbpbdbpbbbdbdddbdbdqbdbpbdqbbb4bd4bbdp7bpqbbd8p7ppbbpbdqddqqqddqddddddqddqdddqq
bdb1d7bpqd3pqppqpbbpbpbdbdqpqdbpbdpqppbpqpqbdbdbddqppqpbdbdqbdqpq6bpbpbdbpqdbpppqqqdqdqdqqdqddqq
pppqppqpppbbbdbddppqpbb4bdbd1pq9dbpbdqddp8pqbpbd6bbpqdbbpppqqpqqppbdqpqpbbpqddqqdddqdqqqdqqqqqq
pq4pbdbbpqpppqpbpqbpbbbbdp8bdbpppppqbppqqpqpbdqb8bd6ppqbp2pqbbbdpppqqqddqdqddqqqqdqdddqddqqqddq
ppbpbbdpqpq3bpbbd2bbpppqbpp41bpqdbbdd3bdqpbppqb8pbdqpbpqbpqddqpqbdppqqpbbddqdqdqqdqqdddqddqdqq
bppqbdqdpppqqppqqqpqpbbpbppbdppbpqppbdqqbdpbbddpbpqbddqqdpbbpbpbppbdpqpbdbdpppqqqqqqdqdqddqqqqqdqddq
bpbdq8ppqbbddpqqd3bppqbbdbpbdpbd2bpqpqd4bbbdbpbdpppbdbdpqbppppqpbbpb6bpqddqddqqqqdqqqqdddqqddqd
pbpqbb6pppbpqpqpbbbp0qppqppbpppqpbbddbbbppqp5bd9bpqbpqdbdpbbbbpbdqddddqdqqdddqqqdqqqdddqdqqqdddq
bppbpbdbbbbpqpqbpqbpppqbbpbdbdb5bdpqb2bpbdqdbppqqdpb76bdbddpbbdpqpppbpqdqqqdqqddqddqqddddddqdqqd
pqbpqpqdpbbddb6bdppbdbpbpqpbd0qppqpbp4bbddpbdqqbdpbpqbbpbb0pqdppbdqq1bpq3bdbdddqdddqdqqdqdqqdq
ppppqpqb6dppb6dbpqdqqbdqppqbpqpqbdbpqbdppqpqbpbbbdbddbddbbdpbbpqpppbdqbdbpbpp7qqdqdqqddqdqdqddqqq
b3ppqbdpqbdpbbpbpqdbpqdpbdppqqq7bdppqbbdpqppqq1pqppqbdbdqbdbppqbdbbbdbppq4pqpqppqqqdddqddqqddqqd
ppqqpqpqbbpqpbbbbddbpqpq8ppqbdp7pqbpqbbdppqpbbdpppbbbpppppbpbdbdqpppbdqqqdqqqqqdddqqqdqqddqqdddqdd
bpb3pbbpppqpbpqdqqbpbdqd5pbbb2ppbbppqpbpbbpqdpqbbbdbpbdqpqdbddpbppbbddbdqqdqddqdqqddqqdddqqddqdqd
pqppbpbdpbpbbpbd7bbbdpbpbpppbdqqbbpqddqdqdbbddbpbdbdbdbdbb6bbpqp7qpbdbdpbpqdqqddddqdqddqddqdqqdqq
pq60pbppbpbdbdppbdqbdqqdqqbbbbpppbppppqpbdpbbpqdbbppb7bdbdpppqpbbppqpqqddqqqdqqdddqqqqqdqqqdddddq
bdbppqpqbbbdbbbbpbppqpbdppqqqbdbbbbpbbbdpbbpbppb6bdpq5pbp6bbbddppqqdqdqdqqdqddqddqddddqdqddddddqd
pqbppqq9dbbdbbbpbdq5b9p1b3ppqqppp6qqbbppbbddbdpbdppqqbppbdbpqbdppbdqp8ppqqqqdqqdqqqddqdqddddd
ppbdbdbpqdbdqpbdpqpq9pq8bbdp22ppqqbppbppqqbpqddqbbddqbbbddbbppppqqqb3ppqpqb06pbdqdqdqddddqdqq
pqpqppbpbbbppbbbbdd1ppbpqd2pbdbpppbbbppqbdq1b9bdpppbpqpbdbbdbddqdbpqpqdqqqddddqqqdqqqddqqdddqdqq
bdpqpqppqqbpqdpbbddqp4pbp3bb4pqbp5bpppqbdqqdpbbpqddbdbppqppqpppqqbpbbbddbdbdppqqdqdqqqdqqdddqdqq
bdbd5bbbppp4ppqq7bbpqbddbpqpbbbbbbbdddbpqbbpb6ppq4pbbp1bdpppqbpqppqqdqqqddqqdqdddddddqddqqqddd
bppq3bdqd8pbdqbdpq40970bpqdppqqbpqbpbp6qdbbpqpqbdbbbdbbpbbppqbppb4bddbb5ddqqdqddqddddddqdd
pqpbpbbddbbp51bdppqpbdbpbpbbbddb09pqppqbpbpbbddpqbbbppqqppqbbbbbdddppqqddqdddqdqdqddqdqdqqqddqdq
bdbdpqbdppqbp6pppbbbpqppqqbbdbdbppbppqppqbpbpppqqppp2pbd8b27pbbpqbdddqdqqqqqdqdqqdqqdddddqqqqdq
bbdbbddbbppbdp0qbdppbpppqbbbppbdbpbppqqbpqddpbbdpbpbpbpbppqpppbbpqddqqqqdqdqdqdqdqqdqqdddqqdqqqqddd
ppqbbddb9bbdpqddpqpppqppqpbpqdbdqqbpqpbbpbpbdbbppqbdq2bbddbpqdp5pqbdqdppbdqqdqbdbpppqqqddqddqdqqq
pppbpb7bdpppqpbppqbpbbbpqpbbpqddb45bdd7bpp9qqpb8pbbpppbpbbpq5pqddqdqqqddqdqdqdddqdqdqqqdqdqqq
pq5ppqqpqbdbdpq506ppqqbpqdp99b3ppbppqqbpqpqbpbbpqddppb8dbdppbp1bbpqp2p3qqddqdqqqqqdddqqdq
bdbdbbpqpqdbpqddpbbdbppbbddpqbbddpqpbbbddbppbpq7ppqqbd4bbd4bbdbpqppp5bpqbbd8ddqqqddddqqddqqqddq
bdppbdbpbbppqqdbbb4ppbdppqqpqqqbbbbdpbdbdppq2pbb45bd1pbdqddpbpppqb8bpp7qqddqqdqqqqdddddddqdqq
pbdqp6pbdqp4bpqbb0bdbddddpp5pqqqbbddppbbd2pbdppbpbpqppqqbbdddbpqpqpb9bbpbdp2qqdddqdqdqqqdqqqq
bbpbbpqbdbbdpqdpqbbd2dbbdbbdbddpbbb7pb5dqbbpqbbddpppqqqbd867bpqpbbppqp29pqqqddqddddddqdddqdd
pbpppbpppbdbdb7pbb5bdb9d9bdbdb5dbppppqqpqpbdqbppbdqpqqpqdbdppbdqbpbdppbdqqbdqdqqqdddqdqqqdqqqdq
bbpppqp9bpbpqdppbbddqqq9bbd9bbpbdbpppqqp7bbppqqdpppqp2bpbpqdbbpbdq00pbdqddqdqqqdqqdqddddqqqdd
pb5bbdpqppbpppbbbdpbd2bpqdb3dpppbbddbdbdq2bppqppbdbbpbppp264pbpqbddqqqqdqddqqqdqqqddqqqdqqddq
p9bbddbdp2bdpqbbpppqbbpbbbbdb7bpqbpqpbdqbpbpppqbbdbpqpbbpbpbd10pbdqqdqddqddqqdqdddddddqddqqddqq
pbppqbp3bd0bbdbddpqqppqppqbpqdqpbpbdbdbpqdppqb1dbpbbddbpppbbdd6pqbppbdqbdqbpbdqddqqqdqdqqdqqdqdq
pbdbpp6qqdqbpqdbbbbbdd75bppqpbpqd3bdqbbddqpqpbpbpbpbdppp3p5qbpqbbpqbb5p6qdddddqqqqdqdqdqdddd
2ppb3pbbdbb4bbbdpqppbpqdbbpqbbbdppbpppppbp1b1ppbbdbpbpppppqqqqqdqddqqdqdqqqqqdqqddddqqdddddqdqq
bpbbdbbddppp9bdqbbbpqbdpbdpbdbpbbbpqpbdqdddbdb0pqdpp1pbbpbbbpbdbdqdp5pbbddqqddqddqqqqdqqdddqqdqd
bdpqbbpqpbbdbpq0bd5pqbbdbppqppbbpbdqdpqdbdpqqpbdqpbbdpqbb7ppbbdbppqqpbbbbdbddddqddqqdddqqqddddqdd
pqpbpqbdbdpqdqpbd7b0pppqqqppqbppbpppqpqqbpqdqbdpqpppbdpbpbd5bb9dbdd1bbddpqpqpqp7qqdqqqqdqqdqdq
bbdppqpbbpqddpp9bd0pbdb4pbpbdppb2p0bbpqddpqqppbppqbppbpbbbbdd4dpqbpqbdddqdqqdqdqqdqqqdqdqqqqqd
b7ppqpqbppppbpbppbdpbbbbddpqbddbppbdpqpqbbddpbdpb82bbb9dpbdpbpbpbbddqdqdqqdddqqqqddqqqdqdqqqqdqd
bpbpbbpqpqdbpbpq0pqbdbppbpppbdpqpqpppbdbppppbdbp26pq2bdbpqdbbbdpbbddqddqdqqqqdqqqqqqdqqddqddqdqd
p4pb38pqbbbbdppbdppbdbbbpqdbbbpqbbdbbbpbpbpqp1bdpppbbddbpqdpqpqppq3qqqqqdqdqdddddddddqqqqddddqq
pqppppp5bdq9pppbdbbpbbpbdppbpbbpbpbpppqbpqbbbpbdpqppqbpqbbppbdqqdddqqddddqqdqdqddqdqqqddqddqqqqqqq
bd8pqpb3bbdddq9pppbd5bbdpbdqb5ddpqbpqbdbbdbdd1bdbb7bdpqppbdqqbpbbpqbpqppqbbd3bddqdddqddddqqq
pbppqq2bdbdppqpbpbppqbbdbdbpppbpqbddqqqpqppbdqpqpbpbdbdqp2bbpqpbpbdbpppqqqpqpqdqdqddqdqqddqdqdqqdq
pbdpbbpqdpqbpqp9b1dppbbpbdbdpqpqbppqpbpqdpbp8bdpqqdppbbddqqqpqpbb41pqbdbdddppqpqqqqqdqddqqqddqq
b8pbdqbdpbpppqb4ppbdbdqp3pbpqbbppqpqpqpqbdbbdpbppbdbpbdpppb1ppbdbppqqdqqdqqqqdqqdqdqdddqqqdqqdqd
pbppbb4bddbbppqppbppqqbbd6dbd4ppbbdpqpqdbbpqpqbbbppbdqqbbbpbdbbddqpqb6dpqbdddddddddqqdqqqdddqqdq
pp2pbdqpbdbbbpqddbbdddqbpbdbbddqbpqdpqdqbdbd6pqpbdqqbdbd4bdppbdqpqbbddbd8bpqbdpqbpbppppqqqqdqddq
pqbbbbddp0bbdbbdbbbpppqpqqpqqddbdppbpbpbdqbppqbdqbddpqbbppbdqpppbpqpqbbbdbbpqdddddqqqqdddqdqqdddqdd
bdpbppqpbppbbbdbbbdppqqbddbbdbbdb8bbb6pppqbd7bpbpbbbddbddqbdpbdpbpppbdqqqdqqdqdqqdddddddddqqdqqdq
pbb2ddqbpqppbdqbbbdpbbbppbbbd0b3b7bdbbpqpqbppbpqdq3bdbbdbdbd3bbbdddpbbbdddqdqdddddddqqdddqddqd
bdbbpp7bppbbppqqpqddq1ppbpqbdppbppp1bpppqbpppbpppq9ppqpqqp6pppp8qqqqqqqdqqqdqqdqqqdqqdqqqdqqdd
ppbpbdpbbddpbppp1pqbpbpbd0pq0bbbdpppbdqqqpqddbdqdbpbdppb5bbpqdbppqq8bdbdbbdddddqqqdqdqqqdqqqdqq
pppppqppppppbppbppbpbpqdpppqb8bpppbbpppqbdpbd2bpbbpqddppbppqqdbdqqqdqqqddqqqddqqqdqqdqqdqqqqqqqqqq
pbdpq7qbdbpqdbdppqqbbddbpqdpbdbpqbbbpbdpbppbdbbpqbd5b6bpqpbdqbpqpppbpbb8bdddqdqqqdddddqqdqqddddq
bdbpbdbpqppqpbdpbpbdbbbppqppqbbbbbddbpbdppbdq3pbbbbbdbbppbbppq2pbdqqddqqddddddqqqddddqqdddqdqqqdqd
b4pqdpppqbdpqpqpqqppbpbbdbbpbbdppp99bdbdbppqbpqdbdqppqqpbbpqdppqpbdqqdpqpqp6pbdqqqdqqqdqdddqdqqq
ppbbdbbpppqpqqbppqq000bbpqbb8b1p6bpqbbdpbdpbpbdpbbpbppb4bpqbdpbdqddqqdqddqqdqqddqddddddqdddqq
bpbbbpb6bb6pbdbbdppppb2pppbdb2ppbbbppbp37p5b9bbbbbdppqbdqdddddqqdqqdddqqdqqqdqqqqdqdddqdddqd
bbppqqpbdbdqdbdppqbdbpqdppqbb9bdbb6bbdbdppbdbpqpbbpqbddbddbbbpqdbbbpqdd6pqpqppbdqqdddqdqqdddddqqd
pbpqdpqpqbbdbdbp77bd2bpbdpbpp9bdpqqpbbdbddbbdbppqppbdqb6pbpqbdbpbdqppqppbdqqqddqdqqddqqdqqdqddq
pbdqbpqppb6pqdbdpqppqqbbpbd5pqbbbbdpqpqdbdpppbpb7pqp6bdqdqpbpqdqbdpqpbpqdpqqpbbddpqqdqqqddqddqqd
p8pq9bdbpppqbbbbdp8bbdp9pppqqpppq3bbbbpqdbpp1pqppbpqppqqpqbdbpbbdbpqddqddqqqqddddqqqqdqdddqqdq
ppqbbbpqbd0bdb0bdpqp3qpbbdbdbbdbpppppqqq7pbdbbb4bpqbpbdbbpqbppp6bbd3dqqqdddqdddddqqqdddqddddq
ppqqpqbpqbddppqqbpqbdpqpppqb4pbpbpbdbppqp93ppqq8bd1ppqqbdbpb2ppqbppqqp8pqbdqdqdqdqqdqdqdqdqqd
pbbdbpqbdpbdqbbbbddpqdbdbpq3ppbpqppbbpppbdbdppp2ppbpb5pqbbbpbdpbbddbdqqddddqdqqqqqqqqddqqdqqddddq
6pbdbd0pppqqqbbppbpbdpqq2pbdbbbddbbdbppbp7bbbpbppqbb5ppqqdpppbppppqqqqdqqqdqdqdddqdqqdddqdqqddq
bp4bdbdqdbdbbddpbbpqdbdbdbdpbbbpbbpq6pbbpppbpbpqdppqpbpqbbpbdbpbpbbpqddqdqdqdddqqqdqqqddqddqdddqdq
pqbdpppbpbbbbbdbdpbbb5bbbp1b79pqpqdbdppqqqdddpqpbdpbdqbdqbpbpqbppqqpqbpqbdb5ddddqddddqddddqdqqq
bd0pbpppbpbpppppqpbppqbbbbbpbbbdpqpbpppqbdqpbbpqdp5bdqpqdppq6bppbdqqdqqqdqddqdddddqdqqqqqdqdqqqdq
bdpqpq5bdbbbbbbdpbd1bdbpbdqp7pp6qqpqpqqdqppqqdpqdbdb7dddpppbdpbpqdpbbd6bdbbp0ppqbdqqdddqqqqqd
bpqd0pbpbpbbpqdbdpq1pp1p8ppp0pp7pqpppbpbpqdppqbpqdqpbbpbdqpbbpbdppqqqddqddqqdqqqqqqqqqqqdqdqdq
b9pqbbbddppbbppqpqqbdbppq0bbdpbppqqb1dbdppppqbb48bpqp0bdpbd2pbbdbbddpbdqdqqqdddqqqdqdqdddqqdd
pbbpqdpbbpqdbdbddbpqb6bdpqbdbpbdqbpqdbdbdppqpbppq5bppbppqbbdpbdpqqbpbp15ppqqqdqddqdqqdqdqqdddqdq
pbpbppbbdbbdbpqdppbbdpqpbpqbbbdpbbdb2bdbdbbbdppp4bdpbdppbpqpqbp9p1qqddqqqqqqddddqdddqdqqddqqdqdq
7bpqpqd1ppqqbdbdbbdppqpbpbppb9pbbdbpbd8ppqbdbbpbdbpqbpbdpbdqqbpbpqdqpbdpqbdqdddqddqqddqdqqdqdqqd
bdbdpbpp5pbpqbdpbdppqqbd2pbbbbdbpqdd3ppbdqqbpbdpbbpqbbp3ppqqpbpqbdbbpppqqqdddqqddddqqdddqqdqqqdq
ppqbbpqpbbbbppbbpqbdppqbb8bppqbdpqqbbbdpqbbp4pqp4ppqpqqbpqdqbdqp9qppqqbbbddddddddddqddqqddddqddq
pbb8bppbdpqb6bdbpqppqpqppqppqbpbdpbbbd9bpqpqbpbpbdpqbpbdqbdbbddbppbppqqdqqddqdqddddqqdqqqddqqdddq
