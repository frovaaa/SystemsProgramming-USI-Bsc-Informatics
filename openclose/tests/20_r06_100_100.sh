#!/bin/bash
exec "$1" "bdpq" < tests/20_r06_100_100.input
