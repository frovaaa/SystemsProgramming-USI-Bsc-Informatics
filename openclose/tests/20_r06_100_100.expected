b454bpb6b7ddqdd
p4bpbpbb1d5dpbppp9pbbbpbppbbpb9dqddqqdqdddqqqqdqqdqdq
pppb7pbppbpppbbpb9dqddqqqdqqdqdqqq
pppb9pbb3dpbpbpp0pbpbppbb1b71dddqqdqdqqqdqdqdqdqqq
b5bbpbbpbbpbbppbbpbppp0qqbbbpbpbbb87dddqdqdddqdqddqqddqddqddqddd
p8qbbppbbbbpppppppbbp1qddqqqqqqqddddqqdd
bpbb9ddqpbb8ddqd
pbpbbbpbbbbppbp2b2dpb2dqqdqqddddqdddqdq
bpbbbbppbb6pbpp69bpbpbp32pbpbb3ddqdqqdqdqdqqdqddqqddddqd
ppppbppbbbppbbbbb8p7qdppbbp5qddqqddddqqdddqqdqqqq
ppppbb4pbb4bpbbpb0dqddqdddqddqqqq
bpp0bppbppb2b9pb2dqbpb9dbp0qdqdddqqdqqdqqd
ppbbbpp8bppppp2p3qqqp3pppppp29p9bpp3qqdqqqqqqqqqqqdqqdddqq
bpb4p32bppbpp2bbbbpbpppp2bbbb9ddddqqqqdqddddqqdqqdqdqd
bpbb0dpbbbpbpbbbb84dp3pbb8bbp8qddddqqdddqdqdddqdqd
ppbbbpbp0qbpbbbbb44bp7bb8p7qddqddddddqddqdddqq
b1d73ppbbpbpp6qqdqddqq
ppppppbppbb419ddp8bp6qdqqqdqqqqqq
4pbbpppbbpbbbp8bppppbb86pbp2qdqddqqqqdqdddqddqqqddq
ppbpb3bpb2bbppbpp413qpbpb8dqdqqdqqdddqddqdqq

b8d3bpbbpp24bbbppppbppppbbpb6dqddqqqqdqqqqdddqqddqd
pbbb6pppbpbbbp0qpppbpppbbbpp59qqdddqqqdqqqdddqdqqqdddq
bppbpbbbbbbppbbpb5b2pb76dqddqddqqddddddqdqqd
pb6ppbpbp0qppbp4qpbbbpbb0d1b3ddqdddqdqqdqdqqdq
pppb6dppb6dqqqpbbpbpbpbbppbpbpp7qqdqdqqddqdqdqddqqq
b3ppbbp7pb1bpbbbp4qdddqddqqddqqd
bbpbbb8pp7qqdddqdd
bpb3pbbp5pbbb2dddqqddqdqd
ppbppbpbbp7bbpbpbb6bbp7qddddqdqddqddqdqqdqq
60pbbbbbpppbpppppbbbppb7dqqdddqqqqqdqqqdddddq
bpbbbbbbpbpbbbbpbbpbbpbppb65pbp6qdqdqqdqddqddqddddqdqddddddqd
b9dbbbb5b9p1b3ppp6qqbbpppbppbpp8qqdqqdqqqddqdqddddd
pp98bp22bbbbpb3pb06dqdqddddqdqq
ppbpbbbppbb1pp2pbpppbbb1b9ddddqqqdqqqddqqdddqdqq
p4pbp3bb4bp5qdddqdqq
5bbbppp47bbpbbbbbbbpb6p4pbbp1qddqqdqdddddddqddqqqddd
bp3qd840970bbpbp6qdbbbbbbpbbpbppb4dbb5ddqqdqddqddddddqdd
pbpbbp51ppbpbpbb09ddqdqdqqqddqdq
pbp6pppbbbbbppbppbpbpppp2p8b27dqqqqqdqdqqdqqdddddqqqqdq
bbbppp0qqqddd
pb9dppbpbbpbpbb2p5qddqdqddqdqqq
pppbpb7pppbpbpbbbpb45d7bpp9qqpb8pbbpppbpbb5ddqdqqqddqdqdqdddqdqdqqqdqdqqq
5506p99b3ppbbbpppb8dppbp1bbp2p3qqddqdqqqqqdddqqdq
pbbpppbbppb74b4bbppp5bb8ddqqqddddqqddqqqddq
ppbpbbbb4bbbpp2pbb451ddpbppb8bpp7qqddqqdqqqqdddddddqdqq
p6p4bbb0dddpp5qqppb2pppbpbpb9bbpp2qqdddqdqdqqqdqqqq
bbpbbb2dbpbbb7pb5dqbb867bpbbpp29qqddqddddddqdddqdd
pbpppbpppb7pbb5b9d9b5dddqdqqqdqqqdq
bbppp9b9b9bbpbpp7bppp2bpbb00ddqdqqqdqqdqddddqqqdd
pb5bppbpppbbp2b3dpp2bpppbbpbppp264qqqdqddqqqdqqqddqqqdqqddq
p9p2bbppbbpbbbb7bbbpbppbbpbbpbp10qdqddqddqqdqdddddddqddqqddqq
pbpbp30qppbppb1dbpbppp6qqqdqdqqdqqdqdq
pbpp6qqdqbbb75bpp3qqpbpbpbpppp3p5qbbbbb5p6qdddddqqqqdqdqdqdddd
2ppb3pbbb4bbppbbbbppbpppppbp1b1dqdqqqqqdqqddddqqdddddqdqq
bpbppp9qbbbppbpb0dpp1pbbpbbp5qddqddqqqqdqqdddqqdqd
bbpbb05bbpppbbb7dddqqqddddqdd
p7b0pbppbppppbp5bb9dd1p7qqdqqqqdqqdqdq
bpppp90pb4pbpppb2p0qppbpbppbpbb4ddqdqqdqdqqdqqqdqdqqqqqd
b7pbppppbpbpppbbppppb82bbb9ddddqqqqddqqqdqdqqqqdqd
bpbpbbpb0bppbppppppbppppbp262qdqqqqdqqqqqqdqqddqddqdqd
p4pb38bbbppppbbbbbbbbbpbpbp1pppp3qqqqqdqdqdddddddddqqqqddddqq
ppppp5q9qqqq
8pb3dq9ppp5bb5ddb1bb7bpbbbpb3dqdddqddddqqq
pb2ppbpbpbbppbp2qdqqddqdqdqqdq
ppbbp9b1dppbbpbpppbp8qdqpbb41ddqqqdqddqqqddqq
b8pbppb4pp3pbbbpbpbppbppppb1dqqqqdqqdqdqdddqqqdqqdqd
pbppbb4dbbpppbb6d4ppbbbbbbbbb6dddddddddqqdqqqdddqqdq
pp2q6q4p8q
bbp0qdd
pbppbppbbbbbb8bbb6pp7qqdddddddddqqdqqdq
pbb2ddqbpbbpbbbppbb0b3b7bbbp3b3dqdddddddqqdddqddqd
bbpp7bp1ppbppbppp1bppbpppbpp9p6pppp8qqqqqqqdqqqdqqdqqqdqqdqqqdqqdd
ppbpppbppp1bpbp00qdbpppb5bb8dddqqqdqdqqqdqqqdqq
ppppppppppbppbppbpppb8bpppbbppp2qqqddqqqddqqqdqqdqqdqqqqqqqqqq
p7qpbbbbppbppbb5b6bbpppbpbb8ddqdqqqdddddqqdqqddddq
bpbpppbpbbbppbbbbpp3pbbbbbbppbbp2qddqqddddddqqqddddqqdddqdqqqdqd
b4dpppbpbbbpbppp99bpp6qqdqqqdqdddqdqqq
ppbbbpb000bbbb8b1p6bbppbppbbpbppb4dqqdqddqqdqqddqddddddqdddqq
bpbbbpb6bb6pbppppb2pppb2ppbbbppbp37p5b9dqqdqqdddqqdqqqdqqqqdqdddqdddqd
bppbb9bb6bppbpbbb6dddqdqqdddddqqd
pbbp772bppbpp9qpbbppb6dqqddqqdqqdqddq
bppb6dbbp5bbpppbpb7p6qdqdqqqddqddqqd
p89bppbbbp8bp9ppp3bbbbpp1qqddddqqqqdqdddqqdq
pbbb0b0p3qpbbbpp7pbbb4bbpbbbppp6b3dqqqdddqdddddqqqdddqddddq
bppb4pbpbpbpp9381bpb2pbp8qdqdqdqqdqdqdqdqqd
pbbbb3ppbppbbpppppp2ppbpb5dqdqqqqqqqqddqqdqqddddq
6p0bbppb2pbbbppbp7bbbpbpbb5ddqdqdddqdqqdddqdqqddq
bp4qdpbpbbbpbb6ddqdddqdq
pppbpbbbbpbbb5bbbp1b79dqdddbpbbbb5ddddqddddqddddqdqqq
0pbpppbpbpppppbpbbbbbpbbpbppbp5qdp6qqqdqddqdddddqdqqqqqdqdqqqdq
5bbbbbp1bp7pp6qqqdqddb7dddpppppb6bbp0qdddqqqqqd
0pbpbpb1pp1p8ppp0pp7qqqqqqqqdqdqdq
b9bppbbbp0bpbb1dpppbb48bp0p2qqdddqqqdqdqdddqqdd
pbpbb6bppbp5bppbpbbpbp15qdqddqdqqdqdqqdddqdq
pbpbppbbppbpbbbpbb2bbppp4pppbbp9p1qqddqqqqqqddddqdddqdqqddqqdqdq
71bppbpbppb9pbbp8qddqdqqdqdqqd
pbpp5pbp2pbb3bppbbbbp3qddddqqdddqqdqqqdq
pbbpbbbbppbbpbb8bbbbbp4p4qqp9qdddddddqddqqddddqddq
pbb8bppb6bpppbppbb9ddqqdqqqddqqdddq
